app/src/main/java/com/heartz/app/data/repositoryimpl/DummyRepositoryImpl.kt

```diff
@@ -0,0 +1,24 @@
+package com.heartz.app.data.repositoryimpl
+
+import com.heartz.app.data.mapper.todata.toData
+import com.heartz.app.data.mapper.todomain.toDomain
+import com.heartz.app.data.service.DummyService
+import com.heartz.app.domain.model.Dummy
+import com.heartz.app.domain.model.DummyResultModel
+import com.heartz.app.domain.repository.DummyRepository
+import javax.inject.Inject
+
+class DummyRepositoryImpl
+@Inject
+constructor(
+    private val dummyService: DummyService
+) : DummyRepository {
+    override suspend fun getDummies(request: Dummy): Result<DummyResultModel> =
+        runCatching {
+            val response =
+                dummyService.getDummies(
+                    request = request.toData()
+                )
```

Comment:
p2
한 줄로 가도 되지 않을까요??
```suggestion
            val response = dummyService.getDummies(request = request.toData())
```